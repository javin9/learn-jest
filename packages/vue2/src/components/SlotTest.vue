<template>
  <div>
    <slot> slot测试默认内容 </slot>
  </div>
</template>