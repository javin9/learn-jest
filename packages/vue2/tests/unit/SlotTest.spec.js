import { shallowMount } from '@vue/test-utils'
import SlotTest from '@/components/SlotTest.vue'

describe('components.SlotTest', () => {
  describe('snapshot', () => {
    it('snapshot测试', () => {
      const wrapper = shallowMount(SlotTest)
      expect(wrapper.vm.$el).toMatchSnapshot()
    })
  })
})