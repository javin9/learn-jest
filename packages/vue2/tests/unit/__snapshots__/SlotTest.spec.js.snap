// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components.SlotTest snapshot snapshot测试 1`] = `
<div>
   slot测试默认内容 
</div>
`;
